src/components/PurchaseOrder/LineListing/LineListing.js

```diff
@@ -22,15 +22,16 @@ class LineListing extends Component {
     const url = match.url;
 
     history.push(`${url}/po-line/view/${meta.id}`);
-  }
+  };
 
   render() {
     const { poLines } = this.props;
     const resultsFormatter = {
       'poLineNumber': item => toString(get(item, ['poLineNumber'], '')),
-      'acquisitionMethod': item => toString(get(item, ['acquisitionMethod'], '')),
-      'owner': item => toString(get(item, ['owner'], '')),
-      'poLineDescription': item => toString(get(item, ['poLineDescription'], '')),
+      'title': item => toString(get(item, 'title', '')),
```

Comment:
IMHO, it's overkill to use `toString` within default value of `_.get()`. I don't think we should expect `item` as null or undefined here, it's always an object, and even if not - stripes will fall in another place.
So maybe it makes sense to write `item.title || ''` here or even just `item.title`
@NikitaSedyx @AndreiShumski2 what do you think?

Reply:
this case it seems we are able skipping toString at all using get from lodash

Reply:
agreed with @aliaksei-chumakou

Reply:
@AndreiKandratsenka  at least we can not introduce it in new code.